DashKitUI/View Models/DashSettingsViewModel.swift

```diff
@@ -112,6 +135,9 @@ class DashSettingsViewModel: ObservableObject {
         basalDeliveryState = pumpManager.status.basalDeliveryState
         basalDeliveryRate = self.pumpManager.basalDeliveryRate
         reservoirLevel = self.pumpManager.reservoirLevel
+        expirationReminderDate = self.pumpManager.state.expirationReminderDate
+        expirationReminderDefault = Int(self.pumpManager.defaultExpirationReminderOffset.hours)
```

Comment:
[nit] include `Offset` in the name?

Also, curious why this comes from the `pumpManager` when the others are pulled from `state`? Should they be consistent?

Reply:
Most are coming from pump manager as they should.  Having the UI reaching into state is a little smelly, imo, but I hate making getters/accessors (more code) that are just boilerplate code.  If you think it's worth making getters/setters for these on pumpManager, lmk.